### PR TITLE
2022/10/8 index.mdx，setup.mdx，welcome-to-nextjs.mdx，editing-the-page.mdx 全數完成

### PR DESCRIPTION
**Pull request by yenbigbear**

### CODE REVIEW

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+```shell
+npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"
+```
+<Blockquote>
+在後台，它使用[`create-next-app`][4]工具，引導您創建 Next.js 應用程式。它通過`--example`參數使用[這個模板][5]。
+
+如果它不起作用，請查看[此頁面][6]。
+</Blockquote>
+
+運行開發服務器
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:37]:
```suggestion
運行開發伺服器
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+- 如果您沒有安裝**Node.js**，請從[此處][1]安裝。您需要 Node.js 版本**10.13**或更高版本。
+- 您將在本教學中使用自己的文字編輯器和終端應用程式。
+
+<Blockquote>
+如果您作業系統使用的是 Windows，我們建議您[下載 Git for Windows][2]並使用它附帶的 Git Bash，它支援本教學中特定於 UNIX 的命令。[適用於 Linux 的 Windows 子系統 (WSL)][3]是另一種選擇。
+</Blockquote>
+
+#### 創建 Next.js 應用
+要創建 Next.js 應用程式，請打開終端，`cd`進入要在其中創建應用式的目錄，然後執行以下指令：
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:26]:
```suggestion
要創建 Next.js 應用程式，請打開終端機，輸入 `cd` 指令來進入要在其中創建應用式的目錄，然後執行以下指令：
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+```shell
+npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"
+```
+<Blockquote>
+在後台，它使用[`create-next-app`][4]工具，引導您創建 Next.js 應用程式。它通過`--example`參數使用[這個模板][5]。
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:32]:
```suggestion
在這個指令的背後，它使用 [`create-next-app`][4] 工具，引導您創建 Next.js 應用程式。它通過`--example` 參數使用[這個模板][5]。
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+如果它不起作用，請查看[此頁面][6]。
+</Blockquote>
+
+運行開發服務器
+您現在有一個名為nextjs-blog. 讓我們cd進入它：
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:38]:
```suggestion
您現在有一個名為 `nextjs-blog` 的資料夾，讓我們使用 `cd` 指令來進入它：
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+```properties
+cd nextjs-blog
+```
+
+然後，運行以下命令：
+
+npm run dev
+這會在端口3000上啟動 Next.js 應用程序的“開發服務器”（稍後會詳細介紹） 。
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:46]:
```suggestion
這會在 3000 端口上啟動 Next.js 應用程式的「開發伺服器」（稍後會詳細介紹） 。
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+如果它不起作用，請查看[此頁面][6]。
+</Blockquote>
+
+運行開發服務器
+您現在有一個名為nextjs-blog. 讓我們cd進入它：
+```properties
+cd nextjs-blog
+```
+
+然後，運行以下命令：
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:43]:
```suggestion
然後，執行以下指令：
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+npm run dev
+這會在端口3000上啟動 Next.js 應用程序的“開發服務器”（稍後會詳細介紹） 。
+
+讓我們檢查一下它是否有效。從瀏覽器打開http://localhost:3000 。
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:48]:
```suggestion
讓我們檢查一下它是否有效。 從瀏覽器打開 `http://localhost:3000` 。
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+[1]: https://nodejs.org/en/ "Node.js安裝"
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:50]:
```suggestion
[1]: https://nodejs.org/en/ "Node.js 安裝"
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+[2]: https://gitforwindows.org/ "Git for Windows下載"
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:51]:
```suggestion
[2]: https://gitforwindows.org/ "Git for Windows 下載"
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+首先，讓我們確保您的開發環境已準備就緒。
+
+- 如果您沒有安裝**Node.js**，請從[此處][1]安裝。您需要 Node.js 版本**10.13**或更高版本。
+- 您將在本教學中使用自己的文字編輯器和終端應用程式。
+
+<Blockquote>
+如果您作業系統使用的是 Windows，我們建議您[下載 Git for Windows][2]並使用它附帶的 Git Bash，它支援本教學中特定於 UNIX 的命令。[適用於 Linux 的 Windows 子系統 (WSL)][3]是另一種選擇。
+</Blockquote>
+
+#### 創建 Next.js 應用
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:25]:
```suggestion
#### 創建 Next.js 應用程式
```

File: pages/learn/basics/create-nextjs-app/setup.mdx [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+
+#### 設置
+首先，讓我們確保您的開發環境已準備就緒。
+
+- 如果您沒有安裝**Node.js**，請從[此處][1]安裝。您需要 Node.js 版本**10.13**或更高版本。
+- 您將在本教學中使用自己的文字編輯器和終端應用程式。
+
+<Blockquote>
+如果您作業系統使用的是 Windows，我們建議您[下載 Git for Windows][2]並使用它附帶的 Git Bash，它支援本教學中特定於 UNIX 的命令。[適用於 Linux 的 Windows 子系統 (WSL)][3]是另一種選擇。
```
Comment (yuaanlin) [inline pages/learn/basics/create-nextjs-app/setup.mdx:22]:
```suggestion
如果您作業系統使用的是 Windows，我們建議您[下載 Git for Windows][2]並使用它附帶的 Git Bash，它支援本教學中特定於 UNIX 的指令。[適用於 Linux 的 Windows 子系統 (WSL)][3]是另一種選擇。
```

**Review by yuaanlin — CHANGES_REQUESTED** [review]

**Comment by yenbigbear** [comment]
2022/10/8 create-nextjs-app/setup.mdx OK

**Review by yuaanlin — APPROVED** [review]
Nice work!

**Pull request merged**